src/metalnx-web/src/main/resources/views/collections/preview.html

```diff
@@ -55,8 +56,8 @@
 			CSV File Preview....
 			<div id="csv" class="preview"></div>
 
-			<button th:onclick="'javascript:save();'">Save</button>
-			<button th:onclick="'javascript:cancel();'">Cancel</button>
+			<button th:if="${permissionType == 'own' and permissionType == 'write'}" th:onclick="'javascript:save();'">Save</button>
+			<button th:if="${permissionType == 'own' and permissionType == 'write'}" th:onclick="'javascript:cancel();'">Cancel</button>
```

Comment:
these should be "or" I think?

Reply:
ahh yes... not sure why I changed them to 'and'. Will get them correct.